pkg/test/resourcefixture/testdata/basic/edgecontainer/v1beta1/edgecontainercluster/create.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: edgecontainer.cnrm.cloud.google.com/v1beta1
+kind: EdgecontainerCluster
+metadata:
+  name: edgecontainercluster-${uniqueId}
+  labels:
+    label-one: "value-one"
+spec:
+  projectRef:
+    external: ${projectId}
+  location: us-central1
+  networking:
+    clusterIpv4CidrBlocks: ["10.0.0.0/16"]
+    servicesIpv4CidrBlocks: ["10.1.0.0/16"]
+  fleet:
+    projectRef: 
+      external: projects/${projectId}
+  authorization:
+    adminUsers:
```

Comment:
Following up my previous comment in service mapping, I do not think we should reference this field to a iam service account. Instead, I think it's ok to provide an active Google account here:
```
 authorization:
    adminUsers:
      username: accountName@google.com
```